classes/Apps/class-admincalendar.php

```diff
@@ -1281,6 +1290,24 @@ public function import_parse_ICS( $ics_file, $calendar, $timezone ) {
 			}
 			return $uploaded;
 		}
+
```

Comment:
In here we need a comment outlining the purpose of this method

`
/**
 * Show and admin notice if a post has a calendar not set error
 * /
`